app/main.py

```diff
@@ -2,0 +1,13 @@
+def copy_file(command: str) -> None:
+    copy_data = command.split()
+
+    if len(copy_data) < 3 or copy_data[0] != "cp":
+        return
+
+    command, file_to_copy_name, new_file_name = copy_data
```

Comment:
The variable `command` is reassigned here, which is unnecessary since it is already used to store the input command. Consider removing this reassignment to avoid confusion.